### ISSUE TITLE
Implement Hybrid Content Telemetry on Disco Pane

### ISSUE THREAD

**Issue by muffinresearch** [mozilla/addons-frontend#4528]
This issue is to implement the [Hybrid Content Telemetry](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/hybrid-content.html#) on the discovery pane.

The library requires a permission - this is being added in: https://bugzilla.mozilla.org/show_bug.cgi?id=1444114 - the work to implement is not necessarily blocked by this though.

Once added this should expose the relevant APIs to the browser. 

* We should look to implement it in a way that mirrors the current GA tracking  - the lib has been designed for this case so that should be straight-forward.
* It should be run in parallel with the google tracking initially and allow for turning each one on or off independently of each other.
* The usage of HCT should be setup with a config var so that observing DNT is configurable based on the implementation. This is because for this implementation which is in-product use (e.g. the disco pane which is shown in Firefox) [HCT will observe telemetry prefs in Firefox](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/collection/hybrid-content.html#js-api). However, should we in the future want to use this on a web property we may want DNT to be the control as to whether data is collected.

**Comment by muffinresearch** [comment]
This is currently blocked on a new incoming package which is ES5 compatible. Once that is shipped the PR can land and then this will need a data collection review before it is enabled in production.

**Comment by muffinresearch** [comment]
Now this is landed the next steps are to test disco and AMO with it off and make sure everything looks good. After that it needs a data review before being enabled. I'll file a new issue to track it's progress.

For now no special QA is needed - other than to check for any breakage around existing analytics events.

**Comment by AlexandraMoga** [comment]
Verified on -dev and stage (AMO and Disco pane) with Firefox 62 and 60esr, Win10x64

Analytics events are logged for actions that are expected to trigger such event.

Also, a new event is present:
```
Not importing the HCT lib: hctEnabled: false
[GA: ON] Not logging to telemetry since hctEnabled: false 
``` 


<details>
<summary>GA events</summary>

[GA] Google Analytics is enabled
Not importing the HCT lib: hctEnabled: false 
[GA: ON] set Object { dimension: "dimension4", value: "TAAR_COHORT_INCLUDED" }
[GA: ON] set Object { dimension: "dimension5", value: "TAAR-LITE-AB" }
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon / Recommendations Shown","eventAction":"recommended_fallback-timeout","eventLabel":"Search by Image"} 
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Installs Started","eventAction":"addon","eventLabel":"Search by Image"} 
[GA: ON] Not logging to telemetry since hctEnabled: false
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Installs","eventAction":"addon","eventLabel":"Search by Image"}
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Uninstalls","eventAction":"addon","eventLabel":"Search by Image"} 
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Installs Started","eventAction":"addon","eventLabel":"Search by Image"}
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Installs","eventAction":"addon","eventLabel":"Search by Image"} 
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Addon Activation","eventAction":"addon","eventLabel":"Search by Image"} 
[GA: ON] Not logging to telemetry since hctEnabled: false 
[GA: ON] sendEvent {"hitType":"event","eventCategory":"AMO Home Hero Experiment / Page View","eventAction":"small"} mo-26128b14c83fe8bec05a.js:1:884230
[GA: ON] Not logging to telemetry since hctEnabled: false

</details>
